oas_docs/output/kibana.serverless.yaml

```diff
@@ -13356,6 +13356,185 @@ paths:
       summary: Health check on Privilege Monitoring
       tags:
         - Security Entity Analytics API
+  /api/entity_analytics/monitoring/users:
+    post:
+      operationId: CreatePrivMonUser
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                is_monitored:
+                  description: Whether the user is actively monitored.
+                  type: boolean
```

Comment:
thinking about it, I think the act of creating them means they are monitored, and the act of (soft) deleting them means they aren't so I wouldn't expect them to be setting the monitoring status via the API. 

which probably means the update API doesn't have any functionality for now, but I think they will be able to set labels through this API so there will be a place for it.